src/App.tsx

```diff
@@ -2,14 +2,25 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
-
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Static list of posts</h1>
-  </div>
-);
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+import { PostList } from './components/PostList/PostList';
+import { Post } from './types/Post';
+
+const preparedPosts: Post[] = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.id) || null,
+  comments: comments.filter(comment => comment.postId === post.id) || [],
```

Comment:
This is really redundant empty array check, filter will return an empty array anyway if no elements will pass the check
```suggestion
  comments: comments.filter(comment => comment.postId === post.id),
```